core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -642,6 +646,25 @@ public Scheme scheme() {
         return scheme;
     }
 
+    @Override
+    @Nullable
+    public String name() {
+        ensureAvailable(RequestLogProperty.NAME);
+        return name;
+    }
+
+    @Override
+    public void name(String name) {
+        requireNonNull(name, "name");
+        checkArgument(!name.isEmpty(), "name is empty.");
```

Comment:
nit: we can combine these two using `isNullOrEmpty`

Reply:
Nope. Can't throw an NPE with it.